user/languages/python.md

```diff
@@ -102,7 +102,7 @@ python:
   - "3.5"
   - "3.6"
   # PyPy versions
-  - "pypy2.7"
+  - "pypy"
   - "pypy3.5"
```

Comment:
could also add
```yaml
  - pypy3.5-5.8.0
  - pypy3.5-5.9.0
  - pypy3.5-5.10.1
```

Reply:
We could but there probably isn't a lot of value in adding many minor versions.